jobbergate-cli/jobbergate_cli/end_to_end_testing/constants.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
+
+
+TEST_APPLICATIONS_PATH = Path("..", "examples")
```

Comment:
Be careful, here. This makes assumption about what directory you are in. It might be better to use `__file__` to make sure you don't have any issues with the relative dir.

Reply:
Yes, this should come from a command-line argument, so the person running the test can customize it.